library/std/src/os/windows/io/handle.rs

```diff
@@ -75,11 +77,11 @@ pub struct OwnedHandle {
 /// `NULL`. This ensures that such FFI calls cannot start using the handle without
 /// checking for `NULL` first.
 ///
-/// This type considers any value other than `NULL` to be valid, including `INVALID_HANDLE_VALUE`.
-/// This is because APIs that use `NULL` as their sentry value don't treat `INVALID_HANDLE_VALUE`
-/// as special.
+/// This type may hold any handle value that [`OwnedHandle`] may hold, except `NULL`. It may
```

Comment:
This sentence sounds like this type cannot hold NULL, but it's the other way around (the type can hold NULL as the sentry value).